### ISSUE TITLE
Chromium 76 results - discussion

### ISSUE THREAD

**Issue by krausest** [krausest/js-framework-benchmark#622]
I just updated results for chrome 76. Overall ranking seems similar, but absolute times have gone up.

Chromium 76: https://krausest.github.io/js-framework-benchmark/2019/table_chrome_76.html
Chromium 75: https://krausest.github.io/js-framework-benchmark/2019/table_chrome_75.html

(I even downgraded Chromium to make sure it wasn't some other update, but got back to the old results)

Feel free to post any observations here.

**Comment by Ploppy3** [comment]
@krausest Chromium 75's url points to the 76 version

**Comment by krausest** [comment]
Thanks - link is fixed

**Comment by ryansolid** [comment]
I sometimes wonder if Chrome goes through multi-release cycles. They seem to go through faster to slow releases in chunks. 67 was fast, 72 was fast, 75 was fast. 68, 73 were really slow. The others in between. I usually wait for fast ones to submit new versions of my libraries if possible.

**Comment by krausest** [comment]
I switched to Ubuntu and Chrome (since a new nvidia driver now allows GPU offloading and nvidia had instructions and xorg packages for ubuntu). All results have been updated.

Numbers look rather similar to manjaro, except script bootup time which now reports much larger values e.g. for domc an ko-jsx. I'm not sure if those are really valid. Any opinions on that?

I had a few runs with an error message from selenium "Server terminated early with status 1". I added some code to retry in this case but had no chance to test whether it works since the issue disappeared right after adding this code...
